README.md

```diff
@@ -107,6 +107,8 @@ interface Options {
   element: HTMLElement;
   // The type of event (defaults to `keydown`)
   trigger: 'keydown' | 'keyup';
+  // Allow input, textarea and select as key event sources (defaults to []). AllowInElement can be 'INPUT', 'TEXTAREA' or 'SELECT'.
```

Comment:
I meant adding this [section](https://github.com/chieffancypants/angular-hotkeys#allowing-hotkeys-in-form-elements) also in our docs :)